### PR TITLE
Update to Zivid SDK v2.1.0

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

**Review by robertkt1 — APPROVED** [review]

**Comment by SatjaSivcev** [comment]
Can you just change the commit message from 'Modify downsample sample to use Zivid API downsample' to 'Modify downsample sample to use Zivid API'? (easier to read)

File: source/Applications/Advanced/Downsample.m [hunk 1]
```diff
@@ -11,13 +11,24 @@
...
 pointCloudOriginal = pointCloud(xyz,'color',rgb);
 
-downsamplingFactor = 4;
-disp(['Downsampling point cloud with factor ',num2str(downsamplingFactor)]);
-[xyzDownsampled,rgbDownsampled] = downsample(xyz,rgb,snr,downsamplingFactor);
+disp(['Size of point cloud before downsampling: ', num2str(pointCloudZivid.Size), ' data points']);
+
+pointCloudZivid.Downsample(Zivid.NET.('PointCloud+Downsampling.By2x2'));
+
+disp(['Size of point cloud before downsampling: ', num2str(pointCloudZivid.Size), ' data points']);
```
Comment (SatjaSivcev) [inline source/Applications/Advanced/Downsample.m:27]: @robertkt1 you already have this on line 23

**Review by SatjaSivcev — COMMENTED** [review]
Minor changes

File: source/Applications/Advanced/Downsample.m [hunk 2]
```diff
@@ -11,13 +11,24 @@
 
 dataFile = [char(System.Environment.GetFolderPath(System.('Environment+SpecialFolder.CommonApplicationData'))),'/Zivid/Zivid3D.zdf'];
 disp(['Reading ZDF frame from file: ',dataFile]);
-[xyz,rgb,snr] = readZDF(dataFile);
+frame = Zivid.NET.Frame(dataFile);
+
+pointCloudZivid = frame.PointCloud();
```
Comment (SatjaSivcev) [inline source/Applications/Advanced/Downsample.m:16]: Why not just pointCloud?

File: source/Applications/Advanced/Downsample.m (hunk 1 above)
Comment (robertkt1) [reply source/Applications/Advanced/Downsample.m:27]: Ah forgot to edit to "size of point cloud after downsampling". To compare the sizes before and after. 
Think it is a nice to have?

File: source/Applications/Advanced/Downsample.m (hunk 2 above)
Comment (SatjaSivcev) [reply source/Applications/Advanced/Downsample.m:16]: I see. Yes that is annoying. Did you check other Matlab samples? Maybe we have it already? If not, I guess this is ok.

Comment (robertkt1) [reply source/Applications/Advanced/Downsample.m:16]: At one other sample it is called ptCloud. Not sure if that is better?

Comment (SatjaSivcev) [reply source/Applications/Advanced/Downsample.m:16]: Yes I think it is. Lets use that for Matlab.

**Pull request merged**